agent/05_agent_create_cluster.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
+
+LOGDIR=${SCRIPTDIR}/logs
+source $SCRIPTDIR/logging.sh
+source $SCRIPTDIR/common.sh
+source $SCRIPTDIR/network.sh
+source $SCRIPTDIR/utils.sh
+source $SCRIPTDIR/validation.sh
+source $SCRIPTDIR/agent/common.sh
+
+early_deploy_validation
+
+function create_image() {
+  # Required until https://issues.redhat.com/browse/AGENT-180 will be implemented
+  pushd ${OCP_DIR}
+
+  $OPENSHIFT_INSTALLER --dir "${OCP_DIR}" --log-level=debug agent create image
```

Comment:
This results in an error -
```
+(./agent/05_agent_create_cluster.sh:20): create_image(): ocp/ostest/openshift-baremetal-install --dir ocp/ostest --log-level=debug agent create image
./agent/05_agent_create_cluster.sh: line 20: ocp/ostest/openshift-baremetal-install: No such file or directory
```
The code is looking for `openshift-baremetal-install` but what we have is `openshift-install`

```
[root@localhost dev-scripts-branches-test]# pwd
/root/dev-scripts-branches-test
[root@localhost dev-scripts-branches-test]# ll ocp/ostest/
total 377692
drwxr-xr-x. 2 root root       172 May 12 13:16 manifests
-rwxr-xr-x. 1 root root 386658304 Apr 21 19:39 openshift-install
-rw-r--r--. 1 root root     95160 May 12 13:16 release_info.json
```

Reply:
Should be overridden in [03_build_agent_installer.sh](https://github.com/openshift-metal3/dev-scripts/pull/1393/files#diff-6e1794f1d1ee5f8fad62e52f1d304a71a79419e856594561a45ccf3efc3708ab), let me check it again